interview-questions.md

```diff
@@ -4,49 +4,77 @@ Answer the following questions. First, without external resources. Challenge you
 
 1. What does CRUD stand for?
 
-  Your answer:
+  Your answer: 
+  Create
+  Read
+  Update
+  Delete
 
   Researched answer:
-
+  "In computer programming, create, read, update, and delete (CRUD) are the four basic functions of persistent storage. Alternate words are sometimes used when defining the four basic functions of CRUD, such as retrieve instead of read, modify instead of update, or destroy instead of delete."
 
 
 2. What are the 5 HTTP verbs?
 
   Your answer:
+  Post
+  Put
+  Get
+  Patch
+  Delete
+  
 
   Researched answer:
-
+"The primary or most-commonly-used HTTP verbs (or methods, as they are properly called) are POST, GET, PUT, PATCH, and DELETE."
 
 
 3. When creating a basic (stateless) class component in React, what are the necessary elements needed to render "Hello World" on the page?
 
   Your answer:
-
+  Without writing out the code completely, we would need AT LEAST the following:
+  An import 
+  A class 
+  A render
+  A return
+  A set of opening and closing divs
+  A text tag like, p, or h1 depending on how youd like it to look. 
+  An export
+  
   Researched answer:
+  .. I dont really have a researched answer. After writing out my answer, I visited multiple websites and found varying answers but (with a few exceptions) mostly all the examples had the above stated elements. 
```

Comment:
thats ok, your answer hit pretty much all of the things we were looking for. it was worded a little strange but we wanted: import, class name + extends, render, return, div/React.Fragment would be a bonus, export